packages/@aws-cdk/aws-lambda-python/README.md

```diff
@@ -44,6 +44,16 @@ If `requirements.txt` or `Pipfile` exists at the entry path, the construct will
 all required modules in a [Lambda compatible Docker container](https://gallery.ecr.aws/sam/build-python3.7)
 according to the `runtime`.
 
+Python bundles are only recreated and published when a file in a source directory has changed. 
```

Comment:
Who generates this lock file and when is it updated? Does the bundling phase of `PythonFunction` do it?

Reply:
This needs to be done by the user